src/components/tooltip/tooltip.e2e.ts

```diff
@@ -12,6 +12,8 @@ describe("calcite-tooltip", () => {
   /**
    * Helps assert the canceled Esc key press when closing tooltips
    * Must be called before the tooltip is closed via keyboard.
+   *
+   * @param page
```

Comment:
Can you roll this file back? I think it was auto-fixed by the linter, but the file is unrelated to this PR.